tests/test_trainer.py

```diff
@@ -73,7 +59,7 @@ def __call__(self, eval_pred):
 
 
 class RegressionModelConfig(PretrainedConfig):
-    def __init__(self, a=0, b=0, double_output=False, **kwargs):
+    def __init__(self, a=0.0, b=0.0, double_output=False, **kwargs):
```

Comment:
There is a `.float()` inside the model so I don't really see what this is adding.

Reply:
Honestly: it removed a few inspection warnings